.gitignore

```diff
@@ -39,3 +39,11 @@ dist/
 
 # Ignore depedencies
 charts/*/charts/*.tgz
+
+# Ignore gotestsum junit reports
```

Comment:
Could we instead have this output to a single well known place? If we consolidated the test task (or perhaps we should have a `ci:test` task in this case) we could output this file to the root of the repo or else where instead of having to micromanage these ignores.

Reply:
My initial plan was to use that, but then it's much easier to create buildkite matrix to test individual package and see where the failure is. Even re-try is much easier as you will not re-run the world (we don't have go test cache). It would be only that particular package.